### PR TITLE
Fix minor GraphQL and Subresource issues

### PR DESCRIPTION
**Pull request by alanpoulain**

### CODE REVIEW

File: core/graphql.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Once enabled, you have nothing to do: your schema describing your API is automat
 
 ## Enabling GraphQL
 
-To enable GraphQL and GraphiQL interface in your API, simply require the graphql-php package using Composer:
+To enable GraphQL and GraphiQL interface in your API, simply require the [graphql-php](https://webonyx.github.io/graphql-php/) package using Composer and clear the cache one more time:
```
Comment (soyuka) [inline core/graphql.md]: why `one more time`?

Comment (alanpoulain) [reply core/graphql.md]: Because `composer require` is doing it once and it has to be done twice in order to work.

Comment (soyuka) [reply core/graphql.md]: mhh okay, in the sentence I guess we can still skip it and just say "clear the cache" no?

Comment (alanpoulain) [reply core/graphql.md]: But the user will wonder why, doesn't it? I thought we could make it explicit.
I could reformulate the sentence like this:
`and clear the cache (it needs to be cleared twice)`

Comment (dunglas) [reply core/graphql.md]: I don't understand why it should be cleared twice. Isn't it mandatory to use `rm -Rf var/cache/*`?

Comment (alanpoulain) [reply core/graphql.md]: No it is not. Clearing the cache twice seems to suffice. I'm not really sure why though. Maybe an issue with the autoloader.

**Comment by dunglas** [comment]
Thanks @alanpoulain (feel free to merge such PRs by yourself btw)

**Pull request merged**